docs/common-BleEmitterUtils.js

```
@@ -0,0 +1,64 @@
+  }
+
+  /**
+   * Emits a command failure event
+   * @param {Error} error - The error that caused the command to fail
+   * @param {Object} context - Additional context about the command that failed
+   */
+  static emitCommandFailure(error, context = {}) {
+    DeviceEventEmitter.emit('LiveDataStatus', 'COMMAND_FAILED');
+    _c_log_h('Command failed:', { error: error.message, ...context });
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)